resources/js/modules/show-hide/index.js

```diff
@@ -1,32 +1,49 @@
-// import $$ from '@utilities/selectors';
-
-// const ShowHide = function ShowHide() {
-//   let content,
-//       chevron;
-
-//   $$.toggleShowHide.forEach(btn => {
-//     btn.addEventListener('click', function(e) {
-//       content = this.nextElementSibling;
-//       content.maxHeight = content.scrollHeight;
-
-//       for ( let i = 0; i < this.childNodes.length; i++ ) {
-//         if (this.childNodes[i].classList.contains('chevron')) {
-//           chevron = this.childNodes[i];
-//         }
-//       }
+import $$ from '@utilities/selectors';
+
+function loopThroughChildren(parent, className) {
```

Comment:
I would be tempted to rename this function. Ideally a function name (or variable name) should tell you its purpose in the application

loopThrough is telling you 'how' the function is doing something, you want to describe 'what' its doing

So, It looks like its setting classnames on a List item? or the UL? maybe call it displayListArrows() or something like that